test/common/README.md

```diff
@@ -56,6 +56,61 @@ symlinks
 ([SeCreateSymbolicLinkPrivilege](https://msdn.microsoft.com/en-us/library/windows/desktop/bb530716(v=vs.85).aspx)).
 On non-Windows platforms, this always returns `true`.
 
+### `collectStream(readable[, callback])`
+
+* return [&lt;Promise>][]
```

Comment:
The return isn’t an array?

Reply:
Not sure I understand, it returns a `string` - the collected 'data' chunks from `readable`.

Reply:
Ah sorry, my reading fail.